### DISCUSSION TITLE
Semi Circle Progress Bar in Chakra UI

### DISCUSSION THREAD

**Discussion by DineshIT** [chakra-ui/chakra-ui#8181] [Q&A]
Hi All,

I'm using the Chakra-UI library for my project. I need to incorporate the Semi circular progress bar. It seems chakra UI has the full circular progress bar but I need something like the below https://classic.yarnpkg.com/en/package/react-semicircle-progressbar

Is there any component available in chakra-ui?

**Comment by zhondori** [comment]
NO. there is no semi-circular progress bar in chakra ui.
